src/gregoriotex/gregoriotex-write.c

```diff
@@ -4701,9 +4701,11 @@ void gregoriotex_write_score(FILE *const f, gregorio_score *const score,
     }
     fprintf(f, "\\GreBeginScore{%s}{%d}{%d}{%d}{%d}{%s}{%u}"
```

Comment:
I'd put a deprecation comment here indicating what the string of arguments will look like after the deprecated ones are removed.

Reply:
OK (also I wrote comments on two struct fields that can be removed).